### ISSUE TITLE
Add a button to change the profile picture

### ISSUE THREAD

**Issue by Rubix982** [Rubix982/Terrabuzz#65]
# FRONTEND
Add a button to the profile picture to change the display picture of the user.

# BACKEND
Not decided in the schema where this picture for the user will go. It seems better to have this be in our MongoDB, but the entity user was decided to reside in SQL.

A good solution would be to make a separate entity RESIDING in SQL that takes the `User ID` as a foreign key, and gives back the profile pictures. A good key article is here https://www.microsoft.com/en-us/research/publication/to-blob-or-not-to-blob-large-object-storage-in-a-database-or-a-filesystem/?from=http%3A%2F%2Fresearch.microsoft.com%2Fapps%2Fpubs%2Fdefault.aspx%3Fid%3D64525.

In short, it suggests to use the database to store images of sizes less than 512 KB, while the ratio for 512 KB - 1 MB are up  to the application developers, while images bigger than 1 MB should reside on the file system.

If we want to implement this, we would have to store the background wallpapers on the file system, and make the profile pictures reside on the database. This is an overhead. 

I checked what the size for Tashik's current picture is by, downloading the image. It's a '.jpg' file with 53.14 KB, which is really reasonable. 

# TODO
ERD is to be modified.

**Comment by Rubix982** [comment]
Not a priority.
